integreat_cms/locale/de/LC_MESSAGES/django.po

```diff
@@ -7688,3 +7706,12 @@ msgstr ""
 
 #~ msgid "Farsi"
 #~ msgstr "Farsi"
+
+#~ msgid "Time Range"
+#~ msgstr "Zeitraum"
+
+#~ msgid "upcoming events"
+#~ msgstr "Zukünftige Termine"
+
+#~ msgid "past events"
+#~ msgstr "Vergangene Termine"
```

Comment:
Please insert your translations in the existing msgid locations, otherwise the translation file cannot be compiled if the same id exists more than once.
In this case, you can fix all your problems with the dev tool `./dev-tools/resolve_translation_conflicts.sh`, so you don't have to manually fix the problems.

Reply:
I have fixed the issues in two commits: one to fix the translation, the other one for fixing pylint and black